__tests__/html/offlineUI.sendFailed.notSend.html

```diff
@@ -42,9 +42,15 @@
 
           clock.tick(400);
 
+          // SETUP: Send 2 messages
           await pageConditions.uiConnected();
-          await pageObjects.sendMessageViaSendBox('42', { waitForSend: false });
+          await pageObjects.sendMessageViaSendBox('First message', { waitForSend: false });
```

Comment:
This test is updated to make sure both "send failed, retry" prompts are not collapsed together.